lib/api/index.js

```diff
@@ -50,4 +52,5 @@ export default {
     remote,
     ipcRenderer,
     shell,
+    options,
```

Comment:
Should this perhaps be named `core`, and contain functions instead of values? I had something like this in mind:

```
import core from './core';

export default {
    bleDriver: bleDriver.api,
    SerialPort,
    logger,
    programming,
    core,
};
```

Then in `lib/api/core/index.js`:

```
import { getAppDir } from '../../util/apps';

export default {
    getAppDir,
};
```